src/helm/proxy/models.py

```diff
@@ -372,6 +372,56 @@ def engine(self) -> str:
         description="GPT-NeoX (20B parameters) autoregressive language model trained on The Pile.",
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, ABLATION_MODEL_TAG, GPTNEO_TOKENIZER_TAG],
     ),
+    Model(
+        group="together",
+        creator_organization="EleutherAI",
+        name="together/pythia-7b",
+        display_name="Pythia (7B)",
+        description="Pythia (7B parameters). The Pythia project combines interpretability analysis and "
+        "scaling laws to understand how knowledge develops and evolves during training in autoregressive "
+        "transformers.",
```

Comment:
Remove these fields to avoid duplicate work.  Just created this PR (https://github.com/stanford-crfm/helm/pull/1567) which you can merge in and help with this.

Reply:
merged it in here!